### PR TITLE
subscribing topic with ros namespace, and setting correct camera topi…

### LINKED ISSUE #5: [Bug] When setting the variable ROS_NAMESPACE the video feed doesn't display.
The error comes from leo.js line 267

### PR DESCRIPTION
**Pull request by Bitterisland6**
…c as video src. Works with updated version of leo_robot repository
closes #5 

Signed-off-by: Bitterisland6 <2010olo@wp.pl>

### CODE REVIEW

**Review by bjsowa — CHANGES_REQUESTED** [review]
The video feed should work even when the `ros_namespace` topic is not published.

**Comment by Bitterisland6** [comment]
Video feed now works with and without topic publishing ros namespace

**Comment by Bitterisland6** [comment]
Changed name of subscribed topic with namespace, to "robot_namespace"

File: js/leo.js [hunk 1]
```diff
@@ -257,6 +273,12 @@ function shutdown() {
     ros.close();
 }
 
+function defaultVideoSrc() {
+    if(typeof robot_namespace == 'undefined')
+        video.src = "http://" + robot_hostname + ":8080/stream?topic=/camera/image_raw&type=ros_compressed";
```
Comment (bjsowa) [inline js/leo.js:281]: We can print some warning to the debug console when we were unable to retrieve the robot namespace and we are assuming it to be `/`

File: js/leo.js [hunk 2]
```diff
@@ -257,6 +273,12 @@ function shutdown() {
...
+}
```
Comment (bjsowa) [inline js/leo.js:283]: Here would be a good place to unsubscribe from the `robot_namespace` topic

**Review by bjsowa — CHANGES_REQUESTED** [review]

File: js/leo.js (hunk 2 above)
Comment (Bitterisland6) [reply js/leo.js:283]: Should we unsubscribe the topic, only when we couldn't get robot namespace, or every time, as the `robot_namespace` topic is latched (should i unsubscribe inside, or outside the `if` statement)?

Comment (bjsowa) [reply js/leo.js:283]: Actually, I think it would be better to do at the start of this function

Comment (Bitterisland6) [reply js/leo.js:283]: Ok, done.

Comment (bjsowa) [reply js/leo.js:283]: We should unsubscribe regardless of whether we got the robot namespace. (We tried for some time but failed, so we give up)

**Review by bjsowa — APPROVED** [review]

**Pull request merged**